core/benchmarks/DiskBlockObjectWriterCloseMethodBenchmark-results.txt

```diff
@@ -0,0 +1,48 @@
+================================================================================================
+emptyDataBuffer
+================================================================================================
+
+OpenJDK 64-Bit Server VM 1.8.0_292-b10 on Mac OS X 11.4
+Apple M1
+emptyDataBuffer:                          Best Time(ms)   Avg Time(ms)   Stdev(ms)    Rate(M/s)   Per Row(ns)   Relative
+------------------------------------------------------------------------------------------------------------------------
+use close                                             0              0           0          0.1       11416.0       1.0X
+use revertPartialWritesAndClose                       0              0           0          0.0       50000.0       0.2X
```

Comment:
Although the else branch is entered only when 'objectsWritten = = 0', there still seem to be some performance differences between the close and revertPartialWritesAndClose method.

I run the bench with ssd，I don't know if there will be a more obvious difference if HDD is used